pkg/monitor/controller/prometheus/images/images.go

```diff
@@ -65,6 +66,7 @@ var versionMap = map[string]Components{
 		PrometheusConfigReloaderWorkload: containerregistry.Image{Name: "prometheus-config-reloader", Tag: "v0.31.1"},
 		PrometheusBeatWorkLoad:           containerregistry.Image{Name: "prometheusbeat", Tag: "6.4.1"},
 		NodeProblemDetector:              containerregistry.Image{Name: "node-problem-detector", Tag: "v0.8.2"},
+		PrometheusAdapter:                containerregistry.Image{Name: "k8s-prometheus-adapter", Tag: "latest"},
 	},
```

Comment:
please don't use `latest` tag,  if have to debug when found PrometheusAdapter issue

Reply:
The recent tag of prometheus-adapter is latest officially，others were out of date；    I will change it to commit number